postgresql/examples/backup.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
```

Comment:
I would suggest to add those patches to generator- so that anyone can generate manifest with/ without the backup.